modules/apps/application-list/application-list-test/src/testIntegration/java/com/liferay/application/list/my/account/permissions/test/PanelAppMyAccountPermissionsTest.java

```diff
@@ -193,7 +193,7 @@ Portlet.class, new TestPortlet(),
 
 	private String _testPortletId;
 
-	private class TestPanelApp extends BasePanelApp {
+	private class TestPanelApp extends BasePortletPanelApp {
```

Comment:
@W2jx This test doesn't actually need to extend `BasePortletPanelApp`, as it's not hard to change the other part of the test to make the portlet it registers accessible by the panel app -- then you can just implement `getPortlet` and return it.